roxie/ccd/ccdcontext.cpp

```diff
@@ -2830,6 +2835,8 @@ class CRoxieServerContext : public CRoxieContextBase, implements IRoxieServerCon
         {
             WorkunitUpdate w(&workUnit->lock());
             w->setState(aborted ? WUStateAborted : (failed ? WUStateFailed : WUStateCompleted));
+            while (clusterNames.ordinality())
```

Comment:
Code was copied from hthor. I suspect it doesn;t really matter since the wu is terminating at this point. but >1 does seem like it would make more sense...